src/main/resources/application.properties

```diff
@@ -1,4 +1,7 @@
 security.oauth2.resource.filter-order=3 
 spring.data.mongodb.host=192.168.99.100
```

Comment:
To IP chyba nie powinno być na repo, bo każdy będzie sobie nawzajem nadpisywał

Reply:
własnie z tym jest problem, są dwie opcje:
stosujemy różne pliki properties i niestety zmieniamy, albo:
wybierzemy wspólnie jakieś IP i każdy z nas skonfiguruje u siebie sprzęt tak, żeby przekierowywać na inne ip, np w pliku hosts.

Reply:
Zrobiłem mały research i widzę że wystarczy dodać do komendy uruchamiającej appkę, w tym przypadku to będzie coś w stylu
```
--spring.data.mongodb.host=192.168.99.100
```
 więc wystarczy że tylko usuniesz tę linijkę, a każdy kto będzie chciał ją uruchomić, po prostu doda sobie IP w konfiguracji uruchamiania (okno Run/Debug Configurations w IntelliJ).

Wyglądałoby to jakoś tak:

![image](https://user-images.githubusercontent.com/19805658/57193864-258b7f80-6f40-11e9-9cfd-e74c29ed5e4a.png)

Chyba warto napisać o tym zmiankę w README.md

Reply:
Ewentualnie można zostawić domyślną wartość jako "localhost", podobnie będzie z właściwością `spring.data.mongodb.port`